.promue2e.yml

```diff
@@ -0,0 +1,24 @@
+go:
```

Comment:
Nit: Maybe we should name this file something like `.promu.e2e.yml`? It looks a bit more readable to me :nerd_face:.

Reply:
Okay, will make this change